LearnositySDK/Utils/JsonObject.cs

```diff
@@ -622,7 +624,7 @@ public string toJson()
 
                     if (this.di.ContainsKey(key))
                     {
-                        sb.Append(this.di[key].ToString());
+                        sb.Append(this.di[key].ToString("R", CultureInfo.InvariantCulture));
```

Comment:
@gonzalozawa why did you do it for integers as well? I think floats should be enough but I may be wrong. Can you give me an example?